.maintain/chaostest/src/config/index.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs')
+const path = require('path')
+const configPath = path.join(__dirname, './config.json')
+
+class Config {
```

Comment:
Am I correctly assuming that `Config` is used to keep state across phases. I.e. that the cleanup logic knows the name of the namespace created in the setup phase?

If so, would you mind adding a doc comment?